nixops/modules/ofborg/log-collector.nix

```diff
@@ -47,7 +47,7 @@ in {
           };
 
           script = ''
-            find "${config.services.ofborg.config_merged.log_storage.path}" -type f -mtime +7 -delete
+            find "${config.services.ofborg.config_merged.log_storage.path}" -type f -mtime +7 -exec sh -c "echo '***DELETED LOGS***' > {}" \;
```

Comment:
If we're going to keep a "dead marker" in logs logs, we might as well also keep the metadata and result files, so the `find` call should ignore `.json` files:

```
[nix-shell:/var/log/ofborg]$ tree ./nixos/nixpkgs.36902
./nixos/nixpkgs.36902
├── 191b09a9-67e8-411c-bad5-ebc1d8760b49
├── 191b09a9-67e8-411c-bad5-ebc1d8760b49.metadata.json
├── 191b09a9-67e8-411c-bad5-ebc1d8760b49.result.json
├── cb80a88d-0615-4413-a720-829a3bc9f3d6
├── cb80a88d-0615-4413-a720-829a3bc9f3d6.metadata.json
├── cb80a88d-0615-4413-a720-829a3bc9f3d6.result.json
├── e4e80a64-efbc-44a7-b9c8-9bdea153eaa6
├── e4e80a64-efbc-44a7-b9c8-9bdea153eaa6.metadata.json
└── e4e80a64-efbc-44a7-b9c8-9bdea153eaa6.result.json

0 directories, 9 files

```

Also, the following `find` line should also be removed, since no dirs will become empty.